config/locales/en.json

```
@@ -479,15 +495,12 @@
 	"dashboard.consent": "has given consent for Service Canada to obtain personal information about them.",
 	"dashboard.view_download_consent": "View their consent",
 	"return_to_dashboard": "Save",
-	"show_consent.title": "Patient Consent",
-	"show_consent.description": "Your patient has indicated the following level of consent:",
-	"show_consent.witness_information": "Witness information",
-	"show_consent.witness_name": "Witness name",
-	"show_consent.witness_signature": "Witness signature",
-	"show_consent.witness_telephone": "Witness telephone",
-	"show_consent.patient_signature": "Patient signature",
-	"show_consent.patient_name": "Patient name",
-	"show_consent.agreement": "<p>Service Canada is authorized under Section 68 and 69 of <em>the Canada Pension Plan (CPP) Regulations</em> to receive personal information (medical and non-medical) about you to determine if you qualify or continue to qualify for CPP disability benefits. Your consent to permit Service Canada to obtain this information is necessary, should Service Canada need this information from persons and organizations listed below.</p><p><strong>I give Service Canada my consent to obtain personal information about me that would help determine if I qualify or continue to qualify for CPP disability benefits. For this reason, Service Canada may contact any of the following persons and organizations if necessary:</strong></p><p><ul><li>medical doctors, nurse practitioners, consultant specialists, or other health care professionals</li><li>educational institutions or other vocational agencies</li><li>my accountant or bookkeeper for information on self-employment</li><li>federal, provincial, territorial, or municipal government departments and agencies</li><li>provincial or territorial workers' compensation boards</li><li>financial institutions (for address updates only)</li><li>medical facilities or hospitals</li><li>administrators of insurance plans</li><li>employers, former employers</li><li>voluntary organizations</li><li>employees (for cases of self-employed persons)</li></ul></p>",
+	"show_consent.title": "'s consent",
```

```suggestion
	"show_consent.title": "'s consent to share medical and personal information",
```
Just to match the Figma, and make it clear what the doc is